assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -83,10 +108,7 @@
   The maximum web console log size has been reached. Use the command-line interface or
   <a href="" ng-click="restartLogs()">reload</a> the log to see new messages.
 </div>
-<div ng-if="!loading && !limitReached && !error" class="text-muted">
+
+<div ng-if="(!loading) && (!limitReached) && (!error) && (!emptyState)" class="text-muted">
   End of log
 </div>
-<div ng-if="error" class="text-muted">
```

Comment:
Might still need this `ng-if="error && !emptyState"` when errors occur halfway through the log.